.env.prod

```diff
@@ -4,3 +4,4 @@ DEBUG_LOGS_ENABLED=0
 STUB_STATUS_GO=0
 ETHEREUM_DEV_CLUSTER=0
 MAINNET_NETWORKS_ENABLED=0
+LOG_LEVEL=error
```

Comment:
We don't want this to be INFO? In case of bug reports etc

Reply:
Are we using `info` anywhere atm?

Reply:
Maybe not yet, but as you say we should. Should also probably sync with what status-go is printing out. Putting it as INFO right now should then include ERROR by default as well (part of logging verbosity scale).

Reply:
changed to `info`